addon/mixins/select-picker.js

```diff
@@ -19,8 +19,8 @@ const emberArrayFunc = function(method) {
   };
 };
 const _contains = emberArrayFunc(
-  // Backwards compatability for Ember < 2.x
-  Ember.Enumerable.keys().indexOf('includes') !== -1 ?
+  // Backwards compatability for Ember < 2.x and > 3.x
+  (typeof ((Ember.A()).includes)) === 'function' ?
```

Comment:
`typeOf` is an Ember function. I was using that over native one. This is a departure from the style used on [line 11](https://github.com/sukima/ember-cli-select-picker/blob/85b1a0aabc4f982ad5b6cadf0ec125fe958c5e5f/addon/mixins/select-picker.js#L11).